clang/include/clang/Basic/Builtins.td

```diff
@@ -4709,6 +4709,18 @@ def HLSLClamp : LangBuiltin<"HLSL_LANG"> {
   let Prototype = "void(...)";
 }
 
+def HLSLCreateHandle : LangBuiltin<"HLSL_LANG"> {
+  let Spellings = ["__builtin_hlsl_create_handle"];
+  let Attributes = [NoThrow, Const];
+  let Prototype = "void*(unsigned char)";
+}
```

Comment:
This looks like an unrelated change sliding back in?

Reply:
It's the result of a merge, I'll remove it.